Makefile

```diff
@@ -1,9 +1,11 @@
 # Some simple testing tasks (sorry, UNIX only).
 
+pytest := python3 -m pytest
```

Comment:
What is intention for this variable? It's basically an alias for just `pytest`, isn't it?